Classes/Domain/Repository/DocumentRepository.php

```diff
@@ -36,7 +36,7 @@
  *
  * @access public
  */
-class DocumentRepository extends Repository
+class DocumentRepository extends \TYPO3\CMS\Extbase\Persistence\Repository
```

Comment:
Which issue is addressed by this change? Since we declare `use TYPO3\CMS\Extbase\Persistence\Repository;` above we shouldn't need to add the namespace here, should we?